metaworld/policies/SawyerButtonPressV1Policy.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+
+from metaworld.policies.action import Action
+from metaworld.policies.policy import Policy, move
+
+
+class SawyerButtonPressV1Policy(Policy):
+
+    @staticmethod
+    def parse_obs(obs):
+        return {
+            'hand_xyz': obs[:3],
+            'button_start_xyz': obs[3:],
+        }
+
+    def get_action(self, obs):
+        o_d = self.parse_obs(obs)
+
+        action = Action({
+            'delta_pos': np.arange(3),
+            'grab_pow': 3
+        })
+
+        action['delta_pos'] = move(o_d['hand_xyz'], to_xyz=self.desired_xyz(o_d), p=4.)
+        action['grab_pow'] = 0.
+
+        return action.array
+
+    @staticmethod
+    def desired_xyz(o_d):
+        pos_curr = o_d['hand_xyz']
+        pos_button = o_d['button_start_xyz']
+        pos_button += np.array([0., 0., -0.07])
+        
+        # align the gripper with the button if the gripper does not have
+        # the same x and z position as the button.
+        hand_x, hand_y, hand_z = pos_curr
+        button_initial_x, button_initial_y, button_initial_z = pos_button
+        if not np.all(np.isclose(np.array([hand_x, hand_z]),
+                                np.array([button_initial_x, button_initial_z]),
+                                atol=0.02)):
+            pos_button[1] = pos_curr[1]
+            return pos_button
```

Comment:
It's somewhat confusing to call this `pos_button` when it's really an amalgamation of `pos_button` and `pos_curr`